libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -144,6 +144,10 @@ class AP_BattMonitor
     /// current_amps - returns the instantaneous current draw in amperes
     bool current_amps(float &current, const uint8_t instance = AP_BATT_PRIMARY_INSTANCE) const WARN_IF_UNUSED;
 
+    /// power watt
+    float power_watts() const;
+    bool power_watts(float &power, const uint8_t instance = AP_BATT_PRIMARY_INSTANCE) const;
```

Comment:
No default to the primary instance, all the primary instance stuff in battery monitor is on the chopping block, so lets not extend it in the meantime.